### PR TITLE
4.0

### PR DESCRIPTION
**Pull request by mattgperry**

### CODE REVIEW

**Comment by hemlok** [comment]
I wonder if we should duplicate the LazyMotion code example to the right here?

Screenshot 2021-03-18 at 11 19 48 am

File: components/Navigation.tsx [hunk 1]
```diff
@@ -34,25 +36,16 @@ const MotionMenu = () => {
...
             />
+            <MenuItem className="guide-upgrade" href="/pages/motion/guide-upgrade.mdx" title="Upgrade guides" />
             <MenuItem
-                className="guide-upgrade-to-framer-motion-2"
-                href="/pages/motion/guide-upgrade-to-framer-motion-2.mdx"
-                title="Upgrade to Motion 2"
-            />
-            <MenuItem
-                className="guide-upgrade-to-framer-motion-3"
-                href="/pages/motion/guide-upgrade-to-framer-motion-3.mdx"
```
Comment (hemlok) [inline components/Navigation.tsx:54]: Should we delete these pages?

**Review by hemlok — APPROVED** [review]
LGTM, comments are nits that could be follow ups

File: components/Navigation.tsx (hunk 1 above)
Comment (mattgperry) [reply components/Navigation.tsx:54]: I was keeping the actual pages to ensure any external links still land there

File: pages/motion/motion-config.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```jsx
+import { motion, MotionConfig } from "framer-motion"
+
+export const MyComponent = ({ isVisible }) => (
+  <MotionConfig transition={{ duration: 1 }}>
+    <motion.div
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+      />
+  </AnimatePresence>
```
Comment (zanona) [inline pages/motion/motion-config.mdx:44]: Perhaps this was meant to be `</MotionConfig>` instead? :wink:

**Pull request merged**